File: spring1-javaspring/src/model/Shift.java
package model;

import java.io.Serializable;
import java.time.LocalDate;
import java.time.LocalTime;

public class Shift implements Serializable {
    private int id;
    private String name;
    private LocalDate date;
    private LocalTime startTime;
    private LocalTime endTime;
    private String description;
    private int numberOfEmployees;
/*

    public Shift(int id, String name, LocalDate date, LocalTime startTime, LocalTime endTime, String description, int numberOfEmployees) {
        this.id = id;
        this.name = name;
        this.date = date;
        this.startTime = startTime;
        this.endTime = endTime;
        this.description = description;
        this.numberOfEmployees = numberOfEmployees;
    }
*/

    public Shift(String name, LocalDate date, LocalTime startTime, LocalTime endTime, String description, int numberOfEmployees) {
        this.name = name;
        this.date = date;
        this.startTime = startTime;
        this.endTime = endTime;
        this.description = description;
        this.numberOfEmployees = numberOfEmployees;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public LocalDate getDate() {
        return date;
    }

    public void setDate(LocalDate date) {
        this.date = date;
    }

    public LocalTime getStartTime() {
        return startTime;
    }

    public void setStartTime(LocalTime startTime) {
        this.startTime = startTime;
    }

    public LocalTime getEndTime() {
        return endTime;
    }

    public void setEndTime(LocalTime endTime) {
        this.endTime = endTime;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getNumberOfEmployees() {
        return numberOfEmployees;
    }

    public void setNumberOfEmployees(int numberOfEmployees) {
        this.numberOfEmployees = numberOfEmployees;
    }

    @Override
    public String toString() {
        return "Shift{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", date=" + date +
                ", startTime=" + startTime +
                ", endTime=" + endTime +
                ", description='" + description + '\'' +
                ", numberOfEmployees=" + numberOfEmployees +
                '}';
    }

}

File: BlazorSep3/model/Shift.cs
using System;
using Newtonsoft.Json;

namespace BlazorSep3.model
{
    public class Shift
    {
        [JsonProperty("id")]
        public int Id { get; set; }
        [JsonProperty("name")] 
        public string Name { get; set; }
        [JsonProperty("date")] 
        public  DateTime Date { get; set; }
        [JsonProperty("startTime")] 
        public DateTime StartTime { get; set; }
        [JsonProperty("endTime")] 
        public DateTime EndTime { get; set; }
        [JsonProperty("description")] 
        public string Description { get; set; }
        [JsonProperty("numberOfEmployees")] 
        public int NumberOfEmployee { get; set; }

        public override string ToString()
        {
            return $"@{Id}: name {Name} date: {Date} start time: {StartTime} end time: {EndTime} description: {Description} n.of employ: {NumberOfEmployee} ";
        }
    }
}